.circleci/config-2_1.yml

```diff
@@ -49,6 +49,7 @@ default_env_vars: &default_env_vars
     # REPEATED_UTEST_TARGET: test-jvm-dtest-some
     # REPEATED_UTEST_TARGET: test-cdc
     # REPEATED_UTEST_TARGET: test-compression
+    # REPEATED_UTEST_TARGET: test-trie
```

Comment:
The new job is nice to have, thanks. Just a detail, we should update the patch file headers (`config-2_1.yml.mid_res.patch` and `config-2_1.yml.high_res.patch`) so the generation script doesn't produce warnings  about hunks:
```
Hunk #1 succeeded at 106 (offset 1 line).
Hunk #2 succeeded at 121 (offset 1 line).
Hunk #3 succeeded at 163 (offset 1 line).
Hunk #4 succeeded at 936 (offset 39 lines).
Hunk #5 succeeded at 950 (offset 39 lines).
...
```
This can be done by running this:
```
cd .circleci

# apply old patches (with hunk messages)
patch -o config-2_1.yml.MIDRES config-2_1.yml config-2_1.yml.mid_res.patch
patch -o config-2_1.yml.HIGHRES config-2_1.yml config-2_1.yml.high_res.patch

# generate new patches
diff -u config-2_1.yml config-2_1.yml.HIGHRES > config-2_1.yml.high_res.patch
diff -u config-2_1.yml config-2_1.yml.MIDRES > config-2_1.yml.mid_res.patch

# verify that the new patches generate the same files and cleanup
./generate.sh -a

```

Reply:
Done